### DISCUSSION TITLE
Companion app not collecting data from Dexcom G6 app

### DISCUSSION THREAD

**Discussion by simondavies560** [NightscoutFoundation/xDrip#2961] [General]
Hi @Navid200 
Starting a new thread here as suggested.

My issue is that the xDrip+ companion app is not receiving BG readings from the Dexcom G6 app, it is just stating "BG Reading Unavailable" (as shown in screenshots). I have followed the instructions, setting all of the notifications and settings as suggested.

I'm using a Pixel 4a (4G) running Android 13, latest version of the Dexcom G6 app, and xDrip version https://github.com/NightscoutFoundation/xDrip/commit/e50dfdfc4f6b47f9dc6448ebca392958e9286352-2-23.05.03.

Units are set to mmol/L in both apps.

Do you have any thoughts on how I might solve this issue?

Thank you very much for your support
Simon

![Screenshot_20230724-182931](https://github.com/NightscoutFoundation/xDrip/assets/36174660/ad5843c2-b9a5-413f-9e68-853e33eb5145)
![Screenshot_20230724-183046](https://github.com/NightscoutFoundation/xDrip/assets/36174660/6a2fe527-c770-456e-ad56-cf3a9f2eabde)
![Screenshot_20230724-183108](https://github.com/NightscoutFoundation/xDrip/assets/36174660/2d24ca06-e62e-4723-b710-e23c9fa6281c)
![Screenshot_20230724-212215](https://github.com/NightscoutFoundation/xDrip/assets/36174660/d2911732-ed2e-4622-8d4d-fd8aa6cec902)
![Screenshot_20230724-183148](https://github.com/NightscoutFoundation/xDrip/assets/36174660/8412a3da-44b8-4097-860f-88feefa72851)

**Comment by jamorham** [comment]
Please update to latest nightly version and also try enabling the companion bluetooth feature.

**Reply by simondavies560** [reply]
Hi @jamorham Thanks for getting in touch. I already had the companion bluetooth feature enabled. I've updated to the 16th July Nightly Build. Unfortunately, it's still stating "BG reading unavailable".
Thanks

**Reply by Navid200** [reply]
Has this been resolved?

**Comment by simondavies560** [comment]
Thank you for checking in. Please close the ticket. I have moved to using
Xdrip+ with CamAPS and the companion functionality is working fine.
